.github/workflows/ci-for-diff-ns.yaml

```
@@ -0,0 +1,49 @@
+name: Plugin test
+on:
+  pull_request:
+
+defaults:
+  run:
+    # reference: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#using-a-specific-shell
+    shell: bash --noprofile --norc -eo pipefail -x {0}
+
+jobs:
+  # This test is required to test latest changes or the changes that not present
+  # with current version of rook-ceph krew plugin
+  with-pr-changes-in-custom-namespace:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: setup cluster
+        uses: ./.github/workflows/cluster-setup
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          custom-ns: "true"
+
+      - name: install script
+        run: sudo install kubectl-rook-ceph.sh /usr/local/bin/kubectl-rook_ceph
+
+      - name: Test Plugin
+        run: |
+          kubectl rook_ceph -o test-operator -n test-cluster  ceph status
```

```suggestion
          kubectl rook_ceph -o test-operator -n test-cluster ceph status
```

same for the lines below